### DISCUSSION TITLE
Template function for making string on exact length by adding space

### DISCUSSION THREAD

**Discussion by yunusemrecatalcam** [grafana/grafana#38282] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
We need a template function for adding spaces to strings so it'll be on an exact length.

**Why is this needed**:
I'm creating a log dashboard and I wanna align all the messages but since some of the string before messages differs on length, I can't do that. I dug into the template functions but couldn't find anything about it, I'm open to any solutions for that. Thanks!

![Screenshot (433)](https://user-images.githubusercontent.com/20454038/118865492-aa5c5280-b8e9-11eb-9062-a4d4c5225378.png)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by hterik** [comment]
Use `line_format "{{printf \"%20s\" .fieldname}}"`. 
Agree it would be useful to have this built in as a function.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
